modules/extensions-support/src/main/java/org/mule/runtime/module/extension/internal/loader/validation/InjectedFieldsModelValidator.java

```diff
@@ -102,7 +102,8 @@ public void visitObject(ObjectType objectType) {
                       validateType(model, type, DefaultEncoding.class);
                     }
                   } catch (Exception e) {
-                    problemsReporter.addWarning(new Problem(model, "Could not validate Class: " + e.getMessage()));
+                    //TODO - This is too verbose and fails always in AST Mode
```

Comment:
ticket? no way to flag when we are in AST mode?

Reply:
I will create a ticket